.lando.base.yml

```diff
@@ -0,0 +1,78 @@
+name: Silta
+recipe: drupal8
+
+config:
+  webroot: web
+  via: nginx
+  php: '7.1'
+  database: mariadb:10.2
+
+tooling:
+  codeception:
+    service: appserver
+    description: Runs Codeception tests
+    cmd: /app/vendor/bin/codecept
+  phpcs:
+    service: appserver
+    description: Runs PHP_CodeSniffer using Drupal, DrupalPractice standards
+    cmd: /app/vendor/bin/phpcs --standard=Drupal,DrupalPractice
+  phpcbf:
+    service: appserver
+    description: Runs PHP Code Beautifier and Fixer using Drupal, DrupalPractice standards
+    cmd: /app/vendor/bin/phpcbf --standard=Drupal,DrupalPractice
+  phpstan:
+    service: appserver
+    description: Runs PHP Static Analysis Tool
+    cmd: /app/vendor/bin/phpstan
+  phpunit:
+    service: appserver
+    description: Runs PHPUnit tests
+    cmd: /app/vendor/bin/phpunit
+  cre:
+    service: appserver
+    description: Runs composer require
+    cmd: composer require
+  cup:
+    service: appserver
+    description: Runs composer update
+    cmd: composer update --with-dependencies
+  cupdc:
+    service: appserver
+    description: Runs composer update drupal/core webflo/drupal-core-require-dev --with-dependencies
+    cmd: composer update drupal/core webflo/drupal-core-require-dev --with-dependencies
+  dsi:
+    service: appserver
+    description: Runs drush site install with drupal8 recipe's --db-url
+    cmd: /app/vendor/bin/drush si --db-url=mysql://drupal8:drupal8@database:3306/drupal8
+
+services:
+  mailhog:
+    type: mailhog
+    hogfrom:
+      - appserver
+
+  appserver:
+    overrides:
+        environment:
+          HASH_SALT: notsosecurehash
+
+  # chrome:
```

Comment:
Having this service in the same file makes a lot of sense.

Reply:
You mean Chrome? We should refactor that as well.

Reply:
A refactor could be useful, but that's for another time.

Reply:
> Having this service in the same file makes a lot of sense.
I see, you were talking about `lando-docker-compose.yml` > `.lando.base.yml` :)